content/locales/fr.ts

```diff
@@ -41,18 +41,98 @@ export const fr = {
+    cancel: "Annuler",
   },
   "/account/profile/user-profile/edit": {
     title: "Informations personnelles",
     breadcrumb: "Profil | modification",
+    usernamePlaceholder: "Saisisser votre nom d'utilisateur",
+    profilePlaceholder: "Saisissez l’URL de votre profil",
+    websitePlaceholder: "Entrez l’URL de votre site Web",
+    picturePlaceholder: "Saisissez l’URL de votre image",
+    zoneInfoPlaceholder: "Saisissez les informations de votre zone",
+    localePlaceholder: "Saisissez votre langue",
+    nameLabel: "Nom complet *",
+    usernameLabel: "Nom d'utilisateur",
+    birthDateLabel: "Date de naissance",
+    profileLabel: "URL de votre profil",
+    websiteLabel: "URL de votre site Web",
+    pictureLabel: "URL de votre image",
+    zoneInfoLabel: "Informations de votre zone",
+    localeLabel: "Votre langue",
+    update: "Mettre à jour mes informations",
+    cancel: "Annuler",
   },
   "/account/profile/addresses/[id]": {
+    title: "Informations personnelles",
+    breadcrumb: "Adresse",
+    primary: "Primaire",
+    update: "Mettre à jour cette adresse",
+    delete: "Supprimer cette adresse",
+    mark: "Utiliser cette adresse comme adresse principale",
+    infoMark:
+      "Vous êtes sur le point de définir cette adresse comme principale.",
+    infoDelete: "Vous êtes sur le point de supprimer cette adresse.",
+    deleteButton: "Supprimer",
+    keep: "Garder cette adresse",
+    confirm: "Confirmer",
+    cancel: "Annuler",
+  },
+  "/account/profile/addresses/[id]/edit": {
     title: "Informations personnelles",
     breadcrumb: "Adresse | modification",
+    streetAddressPlaceholder: "Saisisser votre adresse",
+    streetAddress2Placeholder: "Saisisser votre complement d'adresse",
+    localityPlaceholder: "Saisisser votre localité",
+    postalCodePlaceholder: "Saisisser votre code postal",
+    regionPlaceholder: "Saisisser votre région",
+    countryPlaceholder: "Saisisser votre pays",
+    addressKindPlaceholder: "Saisisser libellé de cette adresse",
+    streetAddressLabel: "Adresse",
+    streetAddress2Label: "Complement d'adresse",
+    localityLabel: "Localité *",
+    postalCodeLabel: "Code postal *",
+    regionLabel: "Région",
+    countryLabel: "Pays *",
+    addressKindLabel: "Libellé",
+    update: "Mettre à jour cette adresse",
+    cancel: "Annuler",
   },
   "/account/profile/addresses/new": {
     title: "Personal information",
     breadcrumb: "Address | création",
+    streetAddressPlaceholder: "Saisisser votre adresse",
+    streetAddress2Placeholder: "Saisisser votre complement d'adresse",
+    localityPlaceholder: "Saisisser votre localité",
+    postalCodePlaceholder: "Saisisser votre code postal",
+    regionPlaceholder: "Saisisser votre région",
+    countryPlaceholder: "Saisisser votre pays",
+    addressKindPlaceholder: "Saisisser libellé de cette adresse",
+    streetAddressLabel: "Adresse",
+    streetAddress2Label: "Complement d'adresse",
+    localityLabel: "Localité *",
```

Comment:
```suggestion
    localityLabel: "Ville *",
```

Reply:
Or keep `"Localité *"` but in the placeholder: `"ex: Paris"`

Reply:
Ville is better imo :)